src/test/java/com/purbon/kafka/topology/integration/SaslPlaintextKafkaContainer.java

```diff
@@ -0,0 +1,142 @@
+package com.purbon.kafka.topology.integration;
+
+import com.github.dockerjava.api.command.ExecCreateCmdResponse;
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.apache.kafka.clients.CommonClientConfigs;
+import org.apache.kafka.clients.admin.Admin;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.images.builder.Transferable;
+import org.testcontainers.utility.DockerImageName;
+
+/** @author <a href="mailto:shh@thathost.com">Sverre H. Huseby</a> */
+final class SaslPlaintextKafkaContainer extends GenericContainer<SaslPlaintextKafkaContainer> {
+
+  private static final DockerImageName DEFAULT_IMAGE =
+      DockerImageName.parse("confluentinc/cp-kafka").withTag("6.0.0");
+  public static final int KAFKA_PORT = 9092;
+  public static final int ZOOKEEPER_PORT = 2181;
+  private static final int PORT_NOT_ASSIGNED = -1;
+  /* Note difference with 0.0.0.0 and localhost: The former will be replaced by the container IP. */
+  private static final String LISTENERS =
+      "SASL_PLAINTEXT://0.0.0.0:9092,INTERNAL://127.0.0.1:29092";
```

Comment:
might be using here the configured kafka port?

Reply:
Yep. Done.